honeybadgermpc/reed_solomon.py

```diff
@@ -32,7 +35,7 @@ def encode_batch(self, data):
         raise NotImplementedError
 
 
-class Decoder(object):
+class Decoder(ABC):
```

Comment:
Does this not require any change in the Encoder/DecoderFactory?

Reply:
None actually-- I don't believe we create any plain `Decoder` objects anywhere.